docs/filter.md

```diff
@@ -26,6 +26,19 @@ Allowed **operators**:
 
 **Value** is a string. All the lookups are case insensitive.
 
+**Escape Sequences** must be used to represent characters in the value that have special meanings in filter, i.e. filter operators.
+
+| Escape Sequence | Represents |
+| -------------- | -------------------- |
+| \\\\            |  \\                   |
+| \\(             |  (                   |
+| \\)             |  )                   |
+| \\&             |  &                   |
+| \\\|             |  \|                   |
+| \\=             |  =                   |
+| \\!             |  !                   |
+| \\~             |  ~                   |
+
```

Comment:
Should we document [utility method](https://github.com/Microsoft/vstest/blob/master/src/Microsoft.TestPlatform.ObjectModel/Utilities/FilterHelper.cs#L23) available for clients here?